Core/GUI.lua

```diff
@@ -1881,40 +1881,44 @@ _G.GameTooltip:HookScript(
 							itemTexture,
 							itemSellPrice = GetItemInfo(v.itemId)
 						if itemLink or itemName or v.name then
-							if not blankAdded and R.db.profile.blankLineBeforeTooltipAdditions then
-								blankAdded = true
-								GameTooltip:AddLine(" ")
-							end
-							local attemptText = " " .. colorize(format(L["(%d/%d attempts)"], v.attempts or 0, v.chance or 0), white)
-							if v.method == CONSTANTS.DETECTION_METHODS.COLLECTION then
-								attemptText = " " .. colorize(format(L["(%d/%d collected)"], v.attempts or 0, v.chance or 0), white)
-							end
-							if v.known or Rarity.db.profile.tooltipAttempts == false then
-								attemptText = ""
-							end
-							GameTooltip:AddLine(
-								colorize(
-									(not rarityAdded and L["Rarity: "] .. (R.db.profile.blankLineAfterRarity and "\n" or "") or "") ..
-										(itemLink or itemName or v.name) .. attemptText,
-									yellow
+							if v.known and R.db.profile.hideKnownItemsInTooltip then
+								GameTooltip:Show()
+							else
```

Comment:
This is the main change. Everything else was just moved inside the `else` block.